src/libraries/System.Configuration.ConfigurationManager/src/README.md

```diff
@@ -0,0 +1,65 @@
+## About
+
+Provides types that support using XML configuration files (`app.config`). This package is mostly useful to assist in porting of the existing .NET Framework code that depends on XML configuration. For a modern configuration API, see [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration/).
```

Comment:
Should we make the deprecation statement stronger? This package only exists to support migration. There is no reason to write new code against this API unless it is in maintenance of code migrated from .NET Framework that already uses it.

Reply:
Updated